[Docs] Add Docs to Explain Airgapped Setup (#7312)
skypilot-org/skypilot: docs/source/cloud-setup/cloud-permissions/aws.rst

      # .bashrc / .zshrc
      # Enable AWS profile named "AWSPowerUserAccess-123456789012"
      export AWS_PROFILE='AWSPowerUserAccess-123456789012'

.. _airgap-aws:

Working with an air-gapped environment
--------------------------------------

:ref:`AWS SSM <aws-ssm>` allows for secure shell access to EC2 instances without direct network access.

> This enables an airgapped setup where launched instances don’t have a public IP address but are still accessible.

Reviewer: ```suggestion
This enables an airgapped setup where launched instances that don’t have public IP addresses can still be accessed.
```

> Assume we have a private VPC for our airgapped AWS cluster with the name ``private-vpc`` and a private security group ``private-sg`` where we want to launch a SkyPilot cluster. We can use a very simple yaml file to enable SkyPilot.

Reviewer: ```suggestion
Given an airgapped AWS cluster with a private VPC ``private-vpc`` and a private security group ``private-sg`, we can use a very simple yaml file to enable SkyPilot.
```
